tools/loc/utils.js

```diff
@@ -126,6 +127,12 @@ export function showButtons(buttonIds) {
   });
 }
 
+export function hideButtons(buttonIds) {
+  buttonIds.forEach((buttonId) => {
+    document.getElementById(buttonId).classList.add('hidden');
```

Comment:
Using a class is fine, but FYI there's also a `hidden` attribute 
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

Reply:
@meganthecoder Nice, thanks! 
Making this change will require some changes to the Loc UI / HTML as well. I will look into handling this separately.